Indicators/QuantConnect.Indicators.csproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
All of the other .csproj files use version 12, which equates to visual studio 2013.  I develop on windows. To prevent unnecessary code churn, please change it back to 12.0.